src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -143,6 +143,8 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
     end
     ```
 
+    > If the route requires an identifier for a specific resource, Lucky will generate a method by the name of that resource `_id`. (e.g. Users::Show generates user_id, and Projects::Show generates project_id)
```

Comment:
This part confused me a bit. Maybe something like this?

“Routed that generate an “id” param will prefix the peak with the underscores version of the resource name. “

And then the examples. Also I think the action and params should be wrapped in ` so it is highlighted as code

Reply:
hmm... Yeah, this one is tough. "peak" loses me here. Maybe
> Routes that require an "id" param will be prefixed with the resource name like `user_id`. 
And then show the examples?

Reply:
Stupid autocorrect. I meant prefix the param. But yeah I like that!

Reply:
Ok, I pushed that up. Let me know if that reads ok. We can keep tweaking if it needs